android/app/src/main/java/com/theoplayer/contentprotectionintegration/integration/keyos/KeyOsWidevineContentProtectionIntegration.kt

```diff
@@ -0,0 +1,20 @@
+package com.theoplayer.contentprotectionintegration.integration.keyos
+
+import com.theoplayer.android.api.source.drm.DRMConfiguration
+import com.theoplayer.android.api.contentprotection.ContentProtectionIntegration
+import com.theoplayer.android.api.contentprotection.LicenseRequestCallback
+import com.theoplayer.android.api.contentprotection.Request
+import java.lang.NullPointerException
+
+class KeyOsWidevineContentProtectionIntegration(private val contentProtectionConfiguration: DRMConfiguration) :
+    ContentProtectionIntegration() {
+    override fun onLicenseRequest(request: Request, callback: LicenseRequestCallback) {
+        if (contentProtectionConfiguration.widevine == null) {
+            throw NullPointerException("The license acquisition URL can not be null")
```

Comment:
Just putting it here to think about:
These parts are always interesting how we should handle them.

NullPointerException (that crashes the app) vs. calling the callback with an error.